src/parser/monk/mistweaver/integrationTests/example.test.ts.snap

```diff
@@ -7174,7 +7174,7 @@ exports[`Mistweaver Monk integration test: example log StatValues matches the st
             <td
               className="text-right"
             >
-              10.21
+              Infinity
```

Comment:
Uh?

Reply:
Yeah. I saw that too but could reproduce it on any logs from the current xpack. I'm assuming it has to do with inflated stats from last xpack and the new stats scaling values

Reply:
I reckon it has to do with a bug

Reply:
In all reality having a leech weight of 10.21 was already insane making me think the test log is wonky

Reply:
Infinity is very possible if the person has way too high secondaries.

Reply:
It doesn't go from 10 to Infinity without a serious bug, weirder when all the bug is supposed to fix is handling "negative" stat values

Reply:
If anything its from the mistweaver code I changed since it only effected mistweaver